tox.ini

```diff
@@ -15,9 +16,10 @@ deps =
     dj18: -rtest_requirements/django_1.8.txt
     dj19: -rtest_requirements/django_1.9.txt
     dj10: -rtest_requirements/django_1.10.txt
+    dj11: -rtest_requirements/django_1.11.txt
     cms32: django-cms>=3.2,<3.3
     cms33: django-cms>=3.3,<3.4
-    cms34: django-cms>=3.4,<3.5
+    cms34: django-cms>=3.4.5,<3.5
```

Comment:
why hardcode 3.4.5 here?

Reply:
To ensure proper 1.11 support (it might be an overkill, though)

Reply:
Yeah I think is fine to leave it as 3.4 because once there's another 3.4.x release we might have to come back and change the ```.5``

Reply:
Done